config/locales/en/course/forum/topics.yml

```diff
@@ -0,0 +1,19 @@
+en:
+  course:
+    forum:
+      topics:
+        topic:
+          started_by: 'Started by '
+          last_post: 'Last Post: '
```

Comment:
Thanks. But the problem is how to generate a link. For example, `Started by: someone`. Someone should be a link. How can I pass a link_to method as the parameter. They will be treated as strings but not html. @lowjoel